recipe/meta.yaml

```
@@ -1,46 +1,53 @@
 {% set name = "wasabi" %}
-{% set version = "0.9.1" %}
-{% set sha256sum = "ada6f13e9b70ef26bf95fad0febdfdebe2005e29a08ad58f4bbae383a97298cf" %}
+{% set version = "1.1.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256sum }}
+  sha256: 4bb3008f003809db0c3e28b4daf20906ea871a2bb43f9914197d540f4f2e0878
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: true  # [py<36]
 
 requirements:
   host:
-    - python
     - pip
-    - setuptools
+    - python
     - wheel
+    - setuptools
   run:
     - python
+    - typing-extensions <5.0.0,>=3.7.4.1  # [py<38]
+    - colorama >=0.4.6  # [win and py>=37]
+
+# jupyter_client.kernelspec.NoSuchKernel: No such kernel named python3
+{% set deselect_tests = " --deselect=tests/test_jupyter.py::test_jupyter" %}
 
 test:
-  requires:
-    - pip 
-    - pytest
   imports:
-    - {{ name }}
+    - wasabi
   commands:
     - pip check
-    - python -m pytest --tb=native --pyargs {{ name }}
-
+    - python -m pytest --tb=native --pyargs wasabi {{ deselect_tests }}
+  requires:
+    - pip 
+    - pytest
 
 about:
   home: http://github.com/ines/wasabi
   license: MIT
   license_family: MIT
   license_file: LICENSE
   summary: A lightweight console printing and formatting toolkit
-
+  description: | 
+    Over the years, I've written countless implementations of coloring and formatting utilities
+    to output messages in our libraries like spaCy, Thinc and Prodigy. While there are many other
+    great open-source options, I've always ended up wanting something slightly different or slightly custom.
```

non-descriptive - please use the very brief description they have on github (same as the summary)

```suggestion
  description: A lightweight console printing and formatting toolkit
```